FirebaseAI/Tests/Unit/GenerativeModelVertexAITests.swift

```diff
@@ -1118,7 +1173,7 @@ final class GenerativeModelVertexAITests: XCTestCase {
       responses += 1
     }
 
-    XCTAssertEqual(responses, 6)
+    XCTAssertEqual(responses, 4)
```

Comment:
Note: This changed in https://github.com/FirebaseExtended/vertexai-sdk-test-data/pull/39.